city_scrapers/spiders/il_sports_facilities_authority.py

```diff
@@ -0,0 +1,105 @@
+from city_scrapers_core.constants import BOARD, COMMITTEE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from dateutil.parser import parse
+
+
+class IlSportsFacilitiesAuthoritySpider(CityScrapersSpider):
+    name = "il_sports_facilities_authority"
+    agency = "Illinois Sports Facilities Authority"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.isfauthority.com/governance/board-meetings/"]
+    location = {
+        "name": "Authority offices",
+        "address": "Guaranteed Rate Field, 333 West 35th Street, Chicago, IL",
+    }
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        self._validate_location(response)
+
+        item = response.css(".wpb_wrapper .inner-text h2::text").getall()[0]
+        meeting = Meeting(
+            title=self._parse_next_title(item),
+            description="",
+            classification=BOARD,
```

Comment:
We might need to check the title here in case the next meeting is a committee meeting, so it could be good to change `_parse_classification` to check a string instead of a `Selector` and use it for upcoming and past meetings

Reply:
Fixed to use `parse_classification` for both past and upcoming meetings in commit bb0d306